From eb6b2a1b0c0893ca6d61fe8cd2ccc868bd21490b Mon Sep 17 00:00:00 2001
From: "Bernhard R. Link" <brlink@debian.org>
Date: Thu, 22 Dec 2016 16:17:56 +0100
Subject: [testsuite] use silent-never instead of never for export in
 buildinfo.test

---
 tests/buildinfo.test | 40 +---------------------------------------
 1 file changed, 1 insertion(+), 39 deletions(-)

diff --git a/tests/buildinfo.test b/tests/buildinfo.test
index 9320522..00053cb 100644
--- a/tests/buildinfo.test
+++ b/tests/buildinfo.test
@@ -151,7 +151,7 @@ rm -r source-a-1 source-b-2 source-c-3
 # first check include:
 mkdir conf
 cat > conf/options <<EOF
-export never
+export silent-never
 EOF
 cat > conf/distributions <<EOF
 Codename: test
@@ -165,8 +165,6 @@ echo Check that include properly ignores a .buildinfo file
 testrun - include test source-a_1_binary.changes 3<<EOF
 stderr
 -v3*=Ignoring buildinfo file: 'source-a_1_all.buildinfo'!
-=Warning: database 'test|main|coal' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/onlyall_1_all.deb')
@@ -218,8 +216,6 @@ mkdir -p pool/main/s/source-a pool/main/s/source-b
 testrun - include test source-b_2_sourceandbuild.changes 3<<EOF
 stderr
 -v3*=Ignoring buildinfo file: 'source-b_2_source.buildinfo'!
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-b/source-b_2.dsc')
@@ -240,8 +236,6 @@ echo Check that include properly ignores a .buildinfo file
 testrun - include test source-a_1_binary.changes 3<<EOF
 stderr
 -v3*=Ignoring buildinfo file: 'source-a_1_all.buildinfo'!
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/onlyall_1_all.deb')
@@ -251,8 +245,6 @@ EOF
 testrun - include test source-b_2_binary.changes 3<<EOF
 stderr
 -v3*=Ignoring buildinfo file: 'source-b_2_abacus.buildinfo'!
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(ofa 'pool/main/s/source-b/onlyany_2_abacus.deb')
 $(opa 'onlyany' '2' 'test' 'main' 'abacus' 'deb')
@@ -261,8 +253,6 @@ EOF
 testrun - include test source-b_2_sourceandbuild.changes 3<<EOF
 stderr
 -v3*=Ignoring buildinfo file: 'source-b_2_source.buildinfo'!
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(ofa 'pool/main/s/source-b/source-b_2.dsc')
 $(ofa 'pool/main/s/source-b/source-b_2.tar.gz')
@@ -272,9 +262,6 @@ EOF
 testrun - include test source-c_3_full.changes 3<<EOF
 stderr
 -v3*=Ignoring buildinfo file: 'source-c_3_abacus.buildinfo'!
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(ofa 'pool/main/s/source-c/source-c_3.dsc')
 $(ofa 'pool/main/s/source-c/source-c_3.tar.gz')
@@ -300,8 +287,6 @@ mkdir -p pool/main/s/source-a pool/main/s/source-b
 echo Check that include properly ignores a .buildinfo file
 testrun - include test source-a_1_binary.changes 3<<EOF
 stderr
-=Warning: database 'test|main|coal' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/onlyall_1_all.deb')
@@ -354,8 +339,6 @@ rm -r pool db
 mkdir -p pool/main/s/source-a pool/main/s/source-b
 testrun - include test source-b_2_sourceandbuild.changes 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-b/source-b_2_source.buildinfo')
@@ -378,8 +361,6 @@ mkdir -p pool/main/s/source-a pool/main/s/source-b pool/main/s/source-c
 echo Check that include properly ignores a .buildinfo file
 testrun - include test source-a_1_binary.changes 3<<EOF
 stderr
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/source-a_1_all.buildinfo')
@@ -390,8 +371,6 @@ EOF
 
 testrun - include test source-b_2_binary.changes 3<<EOF
 stderr
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(ofa 'pool/main/s/source-b/source-b_2_abacus.buildinfo')
 $(ofa 'pool/main/s/source-b/onlyany_2_abacus.deb')
@@ -401,8 +380,6 @@ EOF
 
 testrun - include test source-b_2_sourceandbuild.changes 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(ofa 'pool/main/s/source-b/source-b_2_source.buildinfo')
 $(ofa 'pool/main/s/source-b/source-b_2.dsc')
@@ -412,9 +389,6 @@ EOF
 
 testrun - include test source-c_3_full.changes 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(ofa 'pool/main/s/source-c/source-c_3_abacus.buildinfo')
 $(ofa 'pool/main/s/source-c/source-c_3.dsc')
@@ -446,9 +420,6 @@ rm './i/source-a_1_source.changes'
 echo check to process all .changes at the same time with tracking:
 testrun - processincoming foo 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/source-a_1_all.buildinfo')
@@ -513,9 +484,6 @@ EOF
 echo check to process all .changes at the same time without storing buildinfo:
 testrun - processincoming foo 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/onlyall_1_all.deb')
@@ -571,9 +539,6 @@ rm './i/source-a_1_source.changes'
 
 testrun - processincoming foo 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/onlyall_1_all.deb')
@@ -633,9 +598,6 @@ rm './i/source-a_1_source.changes'
 
 testrun - processincoming foo 3<<EOF
 stderr
-=Warning: database 'test|main|source' was modified but no index file was exported.
-=Warning: database 'test|main|abacus' was modified but no index file was exported.
-=Changes will only be visible after the next 'export'!
 stdout
 $(odb)
 $(ofa 'pool/main/s/source-a/onlyall_1_all.deb')
