From: Hilko Bengen <bengen@debian.org>
Date: Wed, 10 Feb 2021 01:47:23 +0100
Subject: Flush stdout, stderr before calling endhook
Bug-Debian: https://bugs.debian.org/982423

Flush stdout and stderr, otherwise output might be discarded.

--- a/main.c
+++ b/main.c
@@ -4906,6 +4906,8 @@
 	if (snprintf(exitcode, 4, "%u", ((unsigned int)status)&255U) > 3)
 		memcpy(exitcode, "255", 4);
 	sethookenvironment(causingfile, NULL, NULL, exitcode);
+	fflush(stdout);
+	fflush(stderr);
 	argv[0] = endhook,
 	(void)execv(endhook, argv);
 	fprintf(stderr, "Error executing '%s': %s\n", endhook,
